dogfood.py

```diff
@@ -21,34 +21,26 @@
 TARGET_DEMO = "demo"
 TARGET_BRIDGE = "bridge"
 TARGET_GRADLE_PLUGIN = "gradle-plugin"
-TARGET_FLUTTER = "flutter"
 
 REPOSITORIES = {
     TARGET_APP: "datadog-android",
     TARGET_DEMO: "shopist-android",
     TARGET_BRIDGE: "dd-bridge-android",
     TARGET_GRADLE_PLUGIN: "dd-sdk-android-gradle-plugin",
-    TARGET_FLUTTER: "dd-sdk-flutter"
```

Comment:
let's add the comment why flutter doesn't need to be presented in this script

Reply:
I can do that -- should we add a comment about React Native too?

Actually, I don't think @louiszawadzki is using the bridge anymore, should we remove the bridge from this update as well?

Reply:
I think comment will be useful. At least me personally - I will forget quickly why Flutter is not here :)

For the React Native we have a dedicated task to update this script, so it will be done later.